devlib/platform/gem5.py

```diff
@@ -127,6 +128,13 @@ def _start_interaction_gem5(self):
             # telnet connection.
             self.stderr_filename = f
 
+            # Keep track of the statistics log file
+            ret = re.search('--stats-file=(?P<file>\S*)', self.gem5args_args)
```

Comment:
Is there a short-form version of the option (e.g. `-s` or something) that can be used to specify the stats file?